easybuild/easyconfigs/c/causalml/causalml-0.8.0-20200909-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,74 @@
+easyblock = 'PythonBundle'
+
+name = 'causalml'
```

Comment:
Sorry to be picky about names, but `CasualML` or `Casual-ML` seem more fitting here :slightly_smiling_face:

Reply:
it was not chosen by me, there's already one other version available. `causalml` is also the name used on pypi.org.
do you think we should change both or leave it as is?

Reply:
ah yeah, I missed the one already in the tree. Then I would not bother with changing the name, `causalml` is good enough.